client/app/bundles/course/assessment/pages/AssessmentsIndex/ActionButtons.tsx

```diff
@@ -53,6 +58,19 @@ const ActionButtons = (props: ActionButtonsProps): JSX.Element => {
         </Tooltip>
       )}
 
+      {assessment.statisticsUrl && (
```

Comment:
Need to add in a check/test that when statistics component is turned off, this button (and the one in the assessment view page) should not be visible, otherwise it will lead to a 404 page.

Reply:
Done